modules/http4s-server/src/main/scala/io/janstenpickle/trace4cats/http4s/server/Http4sContextConstructors.scala

```diff
@@ -0,0 +1,27 @@
+package io.janstenpickle.trace4cats.http4s.server
+
+import cats.Applicative
+import io.janstenpickle.trace4cats.Span
+import io.janstenpickle.trace4cats.http4s.common.{Http4sHeaders, Http4sRequestFilter, Http4sSpanNamer, Request_}
+import io.janstenpickle.trace4cats.inject.{ContextConstructor, SpanParams}
+import io.janstenpickle.trace4cats.model.SpanKind
+
+object Http4sContextConstructors {
```

Comment:
This name is pretty fine and has some concrete meaning.

But the name `ContextConstructor` seems too narrow compared to its universal form `A => R[B]`.
Maybe we could give it some more general name?

Reply:
Yeah I agree on naming, one of the two hard things in computer science. I'll have a think.